<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>582</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>-1</y>
      <width>901</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Patients</string>
     </attribute>
     <widget class="QLineEdit" name="nomRechercherEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenomRechercherEdit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateFinRechercherEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1900</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="numeroRechercherEdit">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Num√©ro d'identification</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercherPatientButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QTableView" name="patientsTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>881</width>
        <height>441</height>
       </rect>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QCheckBox" name="enableDateFinCheckBox">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateDebutRechercherEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="enableDateDebutCheckBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Planification</string>
     </attribute>
     <widget class="QTreeView" name="personnelTreeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="actionPatient"/>
     <addaction name="actionPersonnel"/>
    </widget>
    <addaction name="menuAjouter"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToolbarAjouter_patient"/>
   <addaction name="actionToolbarAjouter_personnel_de_soins"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionPatient">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionPersonnel">
   <property name="text">
    <string>Personnel de soins</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionToolbarAjouter_patient">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/img/patient-robe.png</normaloff>:/resources/img/patient-robe.png</iconset>
   </property>
   <property name="text">
    <string>Ajouter patient</string>
   </property>
  </action>
  <action name="actionToolbarAjouter_personnel_de_soins">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/img/stethoscope.png</normaloff>:/resources/img/stethoscope.png</iconset>
   </property>
   <property name="text">
    <string>Ajouter personnel de soins</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
